From fa3d7d32ec8b1614d4fc4afe733112b29c5c7593 Mon Sep 17 00:00:00 2001
From: Evangelos Foutras <evangelos@foutrelis.com>
Date: Mon, 27 Apr 2015 11:55:12 +0300
Subject: [PATCH] Update F5 binding for new ThunarWindow::reload signal

In commit b839b29d6dfa57aab9c3cec97eca7624a483a45e (Extend reload
functions to allow a full reload) an extra parameter was added to
reload functions that specifies whether file information should be
reloaded as well.
---
 thunar/thunar-window.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/thunar/thunar-window.c b/thunar/thunar-window.c
index bc51b5e..e8b8b9d 100644
--- a/thunar/thunar-window.c
+++ b/thunar/thunar-window.c
@@ -623,7 +623,7 @@ thunar_window_class_init (ThunarWindowClass *klass)
   /* setup the key bindings for the windows */
   binding_set = gtk_binding_set_by_class (klass);
   gtk_binding_entry_add_signal (binding_set, GDK_BackSpace, 0, "back", 0);
-  gtk_binding_entry_add_signal (binding_set, GDK_F5, 0, "reload", 0);
+  gtk_binding_entry_add_signal (binding_set, GDK_F5, 0, "reload", 1, G_TYPE_BOOLEAN, TRUE);
   gtk_binding_entry_add_signal (binding_set, GDK_F9, 0, "toggle-sidepane", 0);
   gtk_binding_entry_add_signal (binding_set, GDK_F10, 0, "toggle-menubar", 0);
   gtk_binding_entry_add_signal (binding_set, GDK_KP_Add, GDK_CONTROL_MASK, "zoom-in", 0);
-- 
2.3.6

